packages/@react-spectrum/s2/src/TagGroup.tsx

```diff
@@ -519,7 +517,7 @@ const avatarSize = {
 
 function Tag({children, textValue, ...props}: TagProps, ref: DOMRef<HTMLDivElement>) {
   textValue ||= typeof children === 'string' ? children : undefined;
-  let ctx = useSlottedContext(TagGroupContext);
+  let ctx = useContext(InternalTagGroupContext);
```

Comment:
Open for a surprise: https://reactspectrum.blob.core.windows.net/reactspectrum/e1d738d3b452f89444ead0ca392099276b3e9a00/storybook-s2/index.html?path=/story/taggroup--disabled

Reply:
hahahaha, that's amazing. also, it startled me